Apps/Sandcastle/gallery/Bing Maps Labels Only.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Use the split property to only show layers on one side of a slider."
+    />
+    <meta
+      name="cesium-sandcastle-labels"
+      content="Beginner, Tutorials, Showcases"
```

Comment:
```suggestion
      content="Beginner, Showcases"
```

"Tutorials" was intended to be used with website tutorials. Do we have a website tutorial planned for this?